cups/dest.c

```diff
@@ -3502,6 +3502,12 @@ cups_enum_dests(
 
     num_dests = _cupsGetDests(http, IPP_OP_CUPS_GET_PRINTERS, NULL, &dests, type, mask);
 
+    /*
```

Comment:
These changes seem unrelated to making cups-driverd run the driver interface programs in parallel. Please separate your changes (probably each on their own branch in your repository).